sechub-developertools/src/main/java/com/daimler/sechub/developertools/admin/DeveloperAdministration.java

```diff
@@ -350,8 +350,8 @@ public String deletExecutionProfile(String profileId) {
     public String createProject(String projectId, String description, String owner, List<String> whiteListURLs, Map<String, String> metaData) {
         /* @formatter:off */
 		StringBuilder json = new StringBuilder();
-		if (description==null || description.isEmpty()) {
-			description = "description for project "+projectId;
+		if (description==null) {
```

Comment:
Please add some spaces before and after equals "==".

Reply:
done